lib/ubiquo_design/cache_managers/varnish.rb

```diff
@@ -226,7 +228,7 @@ def ban(url, options = {})
           else
             result_url = '^' + Regexp.escape(base_url_without_host) + '/?' + url.last
           end
-          varnish_request('BAN', result_url, host, warmup_url)
+          varnish_request('BAN', result_url, host, warmup_url, !!options.fetch(:apply_on_every_locale, true))
```

Comment:
You need to update the "varnish_request" method signature on this class too to accept the 2 additional parameters. Even if it won't use them.

It very unlekiley that we are going to use this base class standalone ever. But it's better to have it working just in case.